modules/apps/27-interchain-accounts/keeper/relay.go

```diff
@@ -44,18 +44,7 @@ func (k Keeper) createOutgoingPacket(
 		return []byte{}, types.ErrInvalidOutgoingData
 	}
 
-	var msgs []sdk.Msg
```

Comment:
We are doing this check inside `SerializeCosmosTx()`. I can't think of any added value to having it twice.